picard/ui/metadatabox/__init__.py

```diff
@@ -234,6 +208,20 @@ def __init__(self, parent=None):
         self._single_track_album = False
         self.ignore_updates = IgnoreUpdatesContext(on_exit=self.update)
 
+        self.mimedata_helper = MimeDataHelper()
+        self.mimedata_helper.register(
+            self.MIMETYPE_PICARD_TAGS,
+            encode_func=lambda tag_diff: tag_diff.to_json().encode('utf-8'),
+            decode_func=lambda target, mimedata: target._paste_from_json(mimedata))
```

Comment:
Just a minor stylistic comment, we usually wrap last parenthesis to next line and add a comma to last line:

```python
        self.mimedata_helper.register(
            self.MIMETYPE_PICARD_TAGS,
            encode_func=lambda tag_diff: tag_diff.to_json().encode('utf-8'),
            decode_func=lambda target, mimedata: target._paste_from_json(mimedata),
        )
``` 

Same for lines below.

The reason is that it makes easier to spot change if one add or remove a parameter, and it's consistent over the whole Picard code (well, it should).

Reply:
@StevilKnevil can you make changes so we can merge this PR to main PR?